packages/components-native/src/InputText/InputText.tsx

```diff
@@ -400,7 +400,6 @@ function InputTextInternal(
           styleOverride?.inputText,
           loading && loadingType === "glimmer" && { color: "transparent" },
         ]}
-        // @ts-expect-error - does exist on 0.71 and up https://github.com/facebook/react-native/pull/39281
         readOnly={readonly}
```

Comment:
This is the only diff in the bundle output but there are no typescript errors caused by this